### DISCUSSION TITLE
请求增加根据文件夹地址分类的功能

### DISCUSSION THREAD

**Discussion by Wenbobobo** [Planshit/Tai#282] [Ideas]
很多场景下，比如Steam文件夹，工作文件夹会经常有新的exe，每次周末玩完之后都会出现大段的未分类时间，如果可以把文件夹也添加进分类里，根据exe的地址自动归类就好了
![image](https://github.com/Planshit/Tai/assets/78262508/636d78b5-b97d-46ff-b6f8-3894085909ce)

Tai算是咱最喜欢开源软件之一，谢谢作者的贡献(｡･∀･)ﾉﾞ

**Comment by noberumotto** [comment]
收到(｡･ω･｡)

**Comment by noberumotto** [comment]
已在1.5.0.5版本中加入此功能
